kuksa_apps/kanto/README.md

```diff
@@ -0,0 +1,6 @@
+# Publish kuksa.val data via Eclipse Kanto
+// TODO
+# Prerequisites
+// TODO
+# Run the example
+// TODO
```

Comment:
Detailed information must be provided here (also maybe using the demo sketch to visualize the flows).